compiler/version_check.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // Version is the GopherJS compiler version string.
-const Version = "1.16-dev"
+const Version = "1.16.0+go1.16.2"
```

Comment:
Do we want to target the upstream patch version, or just the minor version?  I think in the past, GopherJS always targeted only a minor version. That is, GopherJS 1.12 was expected to work with any patch of Go 1.12.x.  But in theory there could be a patch version that breaks GopherJS, so maybe it's safer to target the patch version?  Does targeting a specific patch version actually do anything (other than advertise support for that patch)?

Reply:
My understanding is that this constant reflects GopherJS version. I think it's a useful signal to show the latest Go version it was tested against, since incompatible changes in standard library internals may happen with patch releases. No a strong opinion though.

Reply:
Works for me.